frameworks/spark/universe/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "2.0",
+  "name": "spark",
+  "version": "{{package-version}}",
+  "maintainer": "support@mesosphere.io",
+  "description": "An spark implementation of a service using the DC/OS SDK",
+  "selected": false,
+  "framework": true,
+  "tags": ["spark"],
+  "postInstallNotes": "DC/OS spark is being installed!\n\n\tDocumentation: https://github.com/mesosphere/dcos-commons/tree/master/frameworks/spark\n\tIssues: https://docs.mesosphere.com/support/",
```

Comment:
`Spark` should be capitalized here and below I believe.  Is `DC/OS Spark` accurate?  I think ASF is picky about Spark being called `Apache Spark`.  Maybe `DC/OS Apache Spark`.